e2e/various-suite/exemplars.spec.ts

```diff
@@ -10,7 +10,7 @@ const addDataSource = () => {
       e2e.components.DataSource.Prometheus.configPage.exemplarsAddButton().click();
       e2e.components.DataSource.Prometheus.configPage.internalLinkSwitch().check({ force: true });
       e2e.components.DataSource.DataSourceHttpSettings.urlInput().type('http://prom-url:9090');
-      e2e.components.DataSourcePicker.inputV2().should('be.visible').click({ force: true });
```

Comment:
I don't know why this broke on this branch, the input exists but is absolutely positioned outside of its parent element so this assertion was failing, I changed it to check for focus after the click, is that sufficient?